adapters/sovrn/sovrntest/supplemental/adunitcode.json

```diff
@@ -20,7 +20,9 @@
           "bidder": {
             "tagid": "123456",
             "adunitcode": "sovrn_auc"
-          }
+          },
+          "gpid": "test-gpid",
+          "adunitcode": "sovrn_auc"
```

Comment:
Perhaps it would be better to set `adunitcode` to a value different from `imp.ext.bidder.adunitcode`?